src/cc/lib/graph/partition.cc

```diff
@@ -176,7 +176,7 @@ void Partition::ReadEdgeIndex(std::filesystem::path path, std::string suffix)
 
     // Extra padding to simplify edge type count calculations.
     m_neighbors_index.back() = m_edge_types.size();
-    m_neighbors_index.push_back(m_edge_types.size());  // Fix for 0 edges in partition
+    m_neighbors_index.push_back(m_edge_types.size()); // Fix for 0 edges in partition
```

Comment:
Please, target main branch.

Reply:
In e2e test I added datasets where one partition has only edges and other partition has only nodes. It fails when .neighbors is run on this dataset.

Reply:
This means it is a bug in the converter, not in C++ code: node map can't contain an internal index pointing to the last element in the neighbor index, because we add an extra element in the close method.

Reply:
Removed, looks like I resolved this when updating theBinaryWriter.add